src/components/Auth/RegisterForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+
+function RegisterForm() {
+  const [inputs, setInputs] = useState({
+    firstname: '',
+    lastname: '',
+    username: '',
+    password: '',
+  });
+
+  const [submitted, setSubmitted] = useState(false);
+
+  const { firstname, lastname, username, password } = inputs;
+
+  const onsubmit = () => {
+    setSubmitted(true);
+  };
+
+  const formChange = (event) => {
+    const { name, value } = event.target;
+    setInputs({ ...inputs, [name]: value });
+  };
+
+  const isEmpty = (obj) => {
+    for (var key in obj) {
+      if (obj[key] === '') return true;
```

Comment:
Let's practice using an early return here. Also, if possible can we use let instead of var